src/core/dbus-execute.c

```diff
@@ -1186,6 +1186,7 @@ const sd_bus_vtable bus_exec_vtable[] = {
         SD_BUS_PROPERTY("PrivateUsersEx", "s", property_get_private_users_ex, offsetof(ExecContext, private_users), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("PrivateMounts", "b", bus_property_get_tristate, offsetof(ExecContext, private_mounts), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("PrivateIPC", "b", bus_property_get_bool, offsetof(ExecContext, private_ipc), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("PrivatePIDs", "b", bus_property_get_bool, offsetof(ExecContext, private_pids), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
I think we should make this an enum right-away, even if we only have on and off initially. But given the history with the other options such as PrivateUsers and ProtectControlgroups I think it would make a ton of sense to start out with a string property and a proper enum right-away, so that we don't have to do this "Ex" property business later.